index.d.ts

```diff
@@ -0,0 +1,136 @@
+declare namespace dargs {
+	interface Options {
+		/**
+		Keys or regex of keys to exclude. Takes precedence over `includes`.
+		*/
+		excludes?: ReadonlyArray<string | RegExp>;
```

Comment:
Since it will be a major release anyway, you can change to `readonly`.

Reply:
Didn't we want to wrap complex types in `Array`/`ReadonlyArray`?

Reply:
So `readonly Array<string | RegExp>` doesn't work?